src/utils/mdast-to-vdom.js

```diff
@@ -223,7 +224,7 @@ class VDOMTransformer {
             const last = stack.pop();
             let html = '';
             for (let j = last; j <= i; j += 1) {
-              html += node.children[j].html || node.children[j].value;
+              html += node.children[j].html || node.children[j].value || toString(node.children[j]);
```

Comment:
From my understanding, this will strip any html in the children.
What about something like:
```js
if (node.children[j].html || node.children[j].value) {
  html += node.children[j].html || node.children[j].value;
}
else {
  const dom = new JSDOM();
  const doc = dom.window.document;
  const frag = toDOM(doc, node.children[j], { fragment: true });
  html += frag.outerHTML;
}
```

The `sanitize` step in the pipeline would still apply after this if I'm not mistaken, so that should also address possible XSS.

Reply:
this, this is a tentative fix... I think the sanitize dom just needs to be smarter. I would like to avoid any reparsing of the html.